self-core-impl/src/main/java/com/selfxdsd/core/BitbucketOrganizationRepos.java

```diff
@@ -0,0 +1,73 @@
+package com.selfxdsd.core;
+
+import com.selfxdsd.api.Repo;
+import com.selfxdsd.api.Repos;
+import com.selfxdsd.api.User;
+import com.selfxdsd.api.storage.Storage;
+
+import java.net.URI;
+import java.util.Iterator;
+import java.util.Spliterator;
+import java.util.function.Consumer;
+
+/**
+ * A Bitbucket Provider Organization Repos.
+ *
+ * @author Nikita Monokov (nmonokov@gmail.com)
+ * @version $Id$
+ * @since 0.0.62
+ */
+public class BitbucketOrganizationRepos implements Repos {
+
+    /**
+     * Organization Repos URI.
+     */
+    private URI reposUri;
+
+    /**
+     * Current authenticated User.
+     */
+    private User owner;
+
+    /**
+     * Bitbucket's JSON Resources.
+     */
+    private JsonResources resources;
+
+    /**
+     * Storage used by Organization Repo.
+     */
+    private Storage storage;
```

Comment:
@nmonokov All attributes of all classes should be final (immutable objects only).